### DISCUSSION TITLE
Touch Screen not working hero 11 naked

### DISCUSSION THREAD

**Discussion by simoneenomis** [gopro/labs#1102] [Q&A]
Hi guys I made my second gopro naked hero 11 but I have a problem that I can't figure out when I connect the rear monitor the touch doesn't work I tried to reinstall the various firmwares even downgrading ... I installed labs .... I even tried to replace the rear monitor with a new one but the touch screen still doesn't work ... so I can't activate the wi-fi ... do you have any idea what it could be? thanks to everyone

**Comment by dnewman-gpsw** [comment]
If you have Labs you can active WiFi via QR code.  
![image](https://github.com/user-attachments/assets/2358503a-7c96-48c9-96f3-882a631dac10)

**Reply by simoneenomis** [reply]
Thanks yes I had done this the Naked is paired but I can't activate the wi-fi to see the previews...

**Reply by dnewman-gpsw** [reply]
If your naked configuration didn't include the WiFi antenna, sometimes the transmitter dies without load. Use !PA to pair to the app uses BLE, which is lower power (antenna not required.)  Via BLE the app will request WiFi on automatically and get the WiFi credentials, so that fact that that is not working, potentially suggest an blow transmitter.

**Reply by simoneenomis** [reply]
please watch my video to see the problem thanks bye

https://drive.google.com/file/d/1nyXKegEsdo3zZoF7n3zu781U0JzsdmQg/view?usp=drivesdk

**Reply by dnewman-gpsw** [reply]
I need to ban google drive, this link is protected.

**Reply by simoneenomis** [reply]
please try again I updated the permissions

**Reply by dnewman-gpsw** [reply]
This could be the antenna is missing in your camera, and the driver circuit is now fried.  Switching 2.4Ghz to 5Ghz and visa versa, does not enable WiFi. That only happens upon viewing media or starting preview. If you don't see GPxxxxxxx in you wifi list, the camera's transmitter might be damaged.

**Reply by simoneenomis** [reply]
Is it possible to replace it?
Isn't it that the wi-fi is only activated/turned on through the touch screen?

**Reply by dnewman-gpsw** [reply]
No, if fried, not replaceable. Some naked camera kits support the antenna, most don't. It is turned on through Quik, like you tried.

**Reply by simoneenomis** [reply]
sorry buddy, hi understood correctly, are there two versions of the motherboard of the Hero 11?

**Reply by dnewman-gpsw** [reply]
Not that I know of.
